setup.py

```diff
@@ -44,9 +44,10 @@ def read_version(package):
 
 tests_require = [
     'decorator>=5,<6',
-    'pytest>=6,<7',
+    'pytest>=6,<9',
     'pytest-cov>=2,<3',
     'testfixtures>=6,<7',
+    'pytest',
```

Comment:
Three lines above the pytest dependency is set to `pytest>=6,<9`, is this unpinned duplicate setting a typo?